express-post-json/server.ts

```diff
@@ -0,0 +1,44 @@
+import express from 'express';
+
+type Grade = {
+  id: number;
+  name: string;
+  course: string;
+  score: number;
+};
+
+type NextId = number;
+
+const grades: Record<number, Grade> = {};
+
+let nextId: NextId = 1;
```

Comment:
If you leave the type off of this completely, TS will figure out that `nextId` must be of type `number`, since `1` is a number.